src/main/java/io/kamax/matrix/client/_MatrixClientRaw.java

```diff
@@ -23,14 +23,17 @@
 import io.kamax.matrix._MatrixID;
 import io.kamax.matrix.hs._MatrixHomeserver;
 
+import java.util.Optional;
+
 public interface _MatrixClientRaw {
 
     MatrixClientContext getContext();
 
     _MatrixHomeserver getHomeserver();
 
-    String getAccessToken();
+    Optional<String> getAccessToken();
 
-    _MatrixID getUser();
+    String getAccessTokenOrThrow();
```

Comment:
Not sure this should be in the interface. While the implementation is useful, seems like bloat for "external" use with a specific exception forced.